src/libs/antares/study/study.h

```diff
@@ -40,7 +40,7 @@
 #include "fwd.h"
 
 #include "simulation.h"
-#include "parameters.h"
```

Comment:
Parameters class is used in the study definition:
I don't think we should remove it, otherwise this header is not self contained anymore (does not compile by itself).